### DISCUSSION TITLE
Airflow 2.3.3 library mismatch

### DISCUSSION THREAD

**Discussion by akashgangulyhf** [astronomer/astronomer-providers#532] [General]
Hello, currently the markupsafe supports (<2.1.0,>=1.1.1) but as per Airflow 2.3.3 [dependency constraints](https://raw.githubusercontent.com/apache/airflow/constraints-2.3.3/constraints-3.10.txt), we need MarkupSafe==2.1.1. Can you please bump up the version check?

**Comment by phanikumv** [comment]
Tracking this in https://github.com/astronomer/astronomer-providers/discussions/528

**Comment by pankajkoti** [comment]
Looks like it was tackled in #528 and hence closing this discussion as it appears stale.
